components/automate-ui/src/app/entities/projects/project.effects.ts

```diff
@@ -196,7 +201,19 @@ export class ProjectEffects {
           observableOf(new GetApplyRulesStatusFailure(error))))));
 
   @Effect()
-  getLatestApplyRulesStatus$ = observableInterval(1000 * POLLING_INTERVAL_IN_SECONDS)
-    .pipe(map(() => new GetApplyRulesStatus()));
+  getLatestApplyRulesStatus$ = observableInterval(1000 * POLLING_INTERVAL_IN_SECONDS).pipe(
+    withLatestFrom(this.store.select(iamMajorVersion).pipe(filter(identity))),
+    withLatestFrom(this.store.select(iamMinorVersion).pipe(filter(identity))),
+    withLatestFrom(this.store.select(allPerms)),
+    filter(([[[_, major], minor], perms]) => {
+      const permObject = perms[ProjectConstants.APPLY_RULES_ENDPOINT];
+      const allowed = permObject && permObject.get;
+      return allowed && major === 'v2' && minor === 'v1';
+    }),
+    switchMap(() =>
+      this.requests.getApplyRulesStatus().pipe(
+        map((resp) => new GetApplyRulesStatusSuccess(resp)),
+        catchError((error: HttpErrorResponse) =>
+          observableOf(new GetApplyRulesStatusFailure(error))))));
```

Comment:
The above block is where the action is happening: every interval collect the data we need from the store--version and permissions--then use a filter to trigger or halt the remaining action, which is to request the rules status.